portfolio/src/main/webapp/script.js

```diff
@@ -112,18 +112,28 @@ window.addEventListener('DOMContentLoaded', (event) => {
     showSlides(slideIndex);
 });
 
-function getMessage() {
-  fetch('/data').then(response => response.json()).then((comment) => {
-
-    const commentElement = document.getElementById('greeting');
-    commentElement.innerHTML = '';
-    commentElement.appendChild(
-        createListElement(comments[0]));
+function getComments() {
+  const limit = document.getElementById('input').value;
+  fetch('/data?limit-input=' + limit).then(response => response.json()).then((comments) => {
```

Comment:
I like how you named the input value to each of the lambda functions with what it is (`response`, `comments`, etc)